scripting/include/get5.inc

```diff
@@ -15,16 +41,1577 @@ enum Get5State {
   Get5State_PostGame,                      // postgame screen + waiting for GOTV to finish broadcast
 };
 
-#define MATCHTEAM_COUNT 4
-
-enum MatchTeam {
-  MatchTeam_Team1,
-  MatchTeam_Team2,
-  MatchTeam_TeamSpec,
-  MatchTeam_TeamNone,
-  MatchTeam_Count,
+enum Get5PauseType {
+  Get5PauseType_None,      // Not paused
+  Get5PauseType_Tech,      // Technical pause
+  Get5PauseType_Tactical,  // Tactical Pause
+  Get5PauseType_Admin,     // Admin/RCON Pause
+  Get5PauseType_Backup     // Special type for match pausing during backups.
 };
 
+methodmap Get5StatusTeam < JSON_Object {
```

Comment:
I find the spacing  here a bit excessive, especially since this is going to be read by any other developer interfacing with this.

Can we remove the blank lines between the class & first method, and the blank link after the last method? 

TBH, I'd be willing to drop the lines between individual methods/properties, but I don't have a very strong opinion. They're just boilerplate, so I would probably lean toward removing the lines.